tools/disable-root-account

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+img_type=$(cat /etc/bios-release.json | grep osimage-img-type)
```

Comment:
Properly, use `JSON.sh` (many examples in this repo) to parse the file. It is guaranteed to be JSON, but not guaranteed to always have current line splitting ;)